crates/sozo/src/ops/migration/mod.rs

```diff
@@ -238,7 +251,7 @@ where
 
                 let _ = TransactionWaiter::new(transaction_hash, migrator.provider())
                     .await
```

Comment:
```suggestion
                 TransactionWaiter::new(transaction_hash, 
                    migrator.provider()).await.map_err(
                       MigrationError::<
                           <SingleOwnerAccount<P, S> as Account>::SignError,
                           <P as Provider>::Error,
                       >::WaitingError,
                )?;
```

might have overlooked this when adding this feature, but this should fix it